csclient/csclient_test.go

```diff
@@ -1,10 +1,11 @@
 // Copyright 2015 Canonical Ltd.
 // Licensed under the LGPLv3, see LICENCE file for details.
 
-package csclient_test // import "gopkg.in/juju/charmrepo.v3/csclient"
+package csclient_test // import "gopkg.in/juju/charmrepo.v4/csclient"
 
 import (
 	"bytes"
+	"context"
```

Comment:
I'm not certain if we should use the built in context or not here. Is it used anywhere else in the project?

Reply:
I think it's OK - juju can use the stdlib context, and this is probably sufficiently in that realm.